<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="newSessionBttn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>New Session</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sessionLogBttn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Session Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timedateBttn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Time and Date</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseBttn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PAUSE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playBttn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBttn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerBttn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>POWER</string>
    </property>
   </widget>
   <widget class="QWidget" name="blueLight" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123);
border-style: solid;
border-width: 5px;
max-width: 8px;
max-height: 8px;
min-width: 8px;
min-height: 8px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="redLight" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123);
border-style: solid;
border-width: 5px;
max-width: 8px;
max-height: 8px;
min-width: 8px;
min-height: 8px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="greenLight" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123);
border-style: solid;
border-width: 5px;
max-width: 8px;
max-height: 8px;
min-width: 8px;
min-height: 8px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="menuText">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot; bgcolor=&quot;#808080&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;MENU&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot; bgcolor=&quot;#808080&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot; bgcolor=&quot;#808080&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeElapsed">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
     </font>
    </property>
    <property name="text">
     <string>0:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="sessionLog">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Monospace'; font-size:10pt; font-weight:400; font-style:normal;&quot; bgcolor=&quot;#808080&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectSensors">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>500</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect Sensors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectSensors">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect Sensors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="condisconPC">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>500</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle PC Connection</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>341</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="PCLog">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>60</y>
      <width>301</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Monospace'; font-size:10pt; font-weight:400; font-style:normal;&quot; bgcolor=&quot;#808080&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>651</x>
      <y>30</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PC View</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="advance">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>500</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Advance</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>471</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Simulation Controls</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>500</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of frames to advance:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="advanceNum">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>960</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>460</y>
      <width>481</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>470</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Simulation Advancement</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="battery">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>547</x>
      <y>410</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>46</red>
          <green>194</green>
          <blue>126</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>46</red>
          <green>194</green>
          <blue>126</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>145</red>
          <green>145</green>
          <blue>145</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="resetPatient">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Patient</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetDevice">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>540</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Device</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>540</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>1 Second = 16 Frames</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>540</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>1 Minute = 960 Frames</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>540</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>1 Frame = 1/16 Seconds</string>
    </property>
   </widget>
   <zorder>textEdit_8</zorder>
   <zorder>textEdit_7</zorder>
   <zorder>textEdit</zorder>
   <zorder>textEdit_3</zorder>
   <zorder>newSessionBttn</zorder>
   <zorder>sessionLogBttn</zorder>
   <zorder>timedateBttn</zorder>
   <zorder>pauseBttn</zorder>
   <zorder>playBttn</zorder>
   <zorder>stopBttn</zorder>
   <zorder>powerBttn</zorder>
   <zorder>blueLight</zorder>
   <zorder>redLight</zorder>
   <zorder>greenLight</zorder>
   <zorder>menuText</zorder>
   <zorder>textEdit_4</zorder>
   <zorder>timeElapsed</zorder>
   <zorder>progressBar</zorder>
   <zorder>dateTimeEdit</zorder>
   <zorder>sessionLog</zorder>
   <zorder>disconnectSensors</zorder>
   <zorder>connectSensors</zorder>
   <zorder>condisconPC</zorder>
   <zorder>textEdit_5</zorder>
   <zorder>PCLog</zorder>
   <zorder>label</zorder>
   <zorder>advance</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>advanceNum</zorder>
   <zorder>label_4</zorder>
   <zorder>battery</zorder>
   <zorder>resetPatient</zorder>
   <zorder>resetDevice</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
